### PR TITLE
Update README with a TOC

### PR DESCRIPTION
**Pull request by bexelbie**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -138,7 +147,7 @@ Install the plugin using:
     vagrant plugin install pkg/<gem name>
 
 
-## Builds
+## Builds <a name="bundler"></a>
```
Comment (navidshaikh) [inline README.md]: s/bundler/builds

**Comment by navidshaikh** [comment]
Also, can you update the CHANGELOG.md as well?

**Comment by LalatenduMohanty** [comment]
LGTM

**Comment by navidshaikh** [comment]
LGTM. Thanks for TOC, the README looks good now.

**Pull request merged**